Configurations/unix-Makefile.tmpl

```diff
@@ -983,6 +983,7 @@ generate_crypto_objects:
 				crypto/objects/obj_mac.num \
 				crypto/objects/obj_xref.txt \
 				> crypto/objects/obj_xref.h )
+	( cd $(SRCDIR);	cat crypto/objects/obj_compat.h >> include/openssl/obj_mac.h )
```

Comment:
Doesn't this mean that anything not on Unix, won't get this?

Reply:
According to https://github.com/openssl/openssl/blob/7165593ce5a07a6860d4d408ad640ee707172936/Configurations/unix-Makefile.tmpl#L941
all these targets are available only on Unix. Am I wrong?

Reply:
I suspect the filename gives away the secret...

Reply:
This PR opens a way to deprecate erroneous names in a normal way.

Reply:
`generate_crypto_objects` is one of the `make update` targets, so really not an issue at all for other platforms.